cli.js

```diff
@@ -69,15 +101,17 @@ function init(files) {
 		];
 	}
 
-	return globby(files).then(function (files) {
-		files.forEach(function (file) {
-			run(path.resolve(file));
-		});
+	return globby(files)
+		.then(function (files) {
+			// convert relative paths to absolute
+			files = files.map(function (file) {
+				return path.join(process.cwd(), file);
+			});
 
```

Comment:
So one function (`handlePaths()`?) that globs the files, filters out js files (like `run` does now), stats.isDirectory() (like `run` does now), convert paths to absolute, then returns them. This function should be used here.